main.go

```diff
@@ -58,50 +57,56 @@ func run(args []string) error {
 	case "new":
 		err := newCommand.Parse(args[2:])
 		if err != nil {
-			return err
+			return "", err
 		}
-		message, commandErr = command.NewCommand(ctx, *newProviderType, path, *newToken, *newGroup, *newApp, *newTag)
+		message, commandErr = withCopyOfWorkTree(newPath, func(workTreeCopy string) (string, error) {
+			return command.NewCommand(ctx, *newProviderType, workTreeCopy, *newToken, *newGroup, *newApp, *newTag)
+		})
```

Comment:
I like it! It took a few more seconds to understand the flow of the code than before, but most likely due to the formatting of the changes in GitHub more than anything else.

Reply:
Yes, there is a readability argument for the JavaScript et al style where you can have a single statement return a value without the return keyword.